From 061738280a8628ab9b13f827b3217a375c40ba31 Mon Sep 17 00:00:00 2001
From: Amit Sunil Dhamne <amit.sunil.dhamne@xilinx.com>
Date: Tue, 8 Sep 2020 17:53:29 -0700
Subject: [PATCH 1575/1851] firmware: xilinx: sync firmware pm_query_id entries
 with PLM query id entries

commit 8ed8a8b53901bcb912aa89366b7231fa721838d6 from
https://github.com/Xilinx/linux-xlnx.git

Sync firmware Query Id enumeration present in xlnx-zynqmp.h with
query Ids supported in PLM firmware. This includes addition of
an additional query to get PLDevice parent

Signed-off-by: Amit Sunil Dhamne <amit.sunil.dhamne@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 include/linux/firmware/xlnx-zynqmp.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/firmware/xlnx-zynqmp.h b/include/linux/firmware/xlnx-zynqmp.h
index 3b0ec22aaa3e..0b5f88c97953 100644
--- a/include/linux/firmware/xlnx-zynqmp.h
+++ b/include/linux/firmware/xlnx-zynqmp.h
@@ -199,6 +199,7 @@ enum pm_query_id {
 	PM_QID_PINCTRL_GET_PIN_GROUPS,
 	PM_QID_CLOCK_GET_NUM_CLOCKS,
 	PM_QID_CLOCK_GET_MAX_DIVISOR,
+	PM_QID_PLD_GET_PARENT,
 };
 
 enum zynqmp_pm_reset_action {
-- 
2.31.1

